utils/mksh/patches/100-dot_mkshrc

```diff
@@ -14,30 +14,23 @@ Subject: [PATCH] Make default mkshrc file suitable for OpenWrt environment:
 Signed-off-by: Alif M. A. <alive4ever at live.com>
 --- a/dot.mkshrc
 +++ b/dot.mkshrc
-@@ -28,8 +28,9 @@
- *) return 0 ;;
- esac
+@@ -56,9 +56,10 @@
+ 	done
+ )
  
--PS1='#'; (( USER_ID )) && PS1='$'; \: "${TERM:=vt100}${HOSTNAME:=$(\ulimit -c \
--    0; hostname 2>/dev/null)}${EDITOR:=/bin/ed}${USER:=$(\ulimit -c 0; id -un \
-+PS1='#'; (( USER_ID )) && PS1='$'; \: "${HOSTNAME:=$(</proc/sys/kernel/hostname\
-+    )} = *([     ]|localhost) && HOSTNAME=$(\ulimit -c \
-+    0; hostname 2>/dev/null)}${EDITOR:=/bin/vi}${USER:=$(\ulimit -c 0; id -un \
-     2>/dev/null || \echo \?)}${MKSH:=$(\builtin whence -p mksh)}"
- HOSTNAME=${HOSTNAME%%*([	 ]).*}; HOSTNAME=${HOSTNAME##*([	 ])}
- [[ $HOSTNAME = ?(ip6-)localhost?(6) ]] && HOSTNAME=
-@@ -52,7 +53,7 @@
- \alias l='ls -F'
- \alias la='l -a'
- \alias ll='l -l'
--\alias lo='l -alo'
-+\alias lo='l -al'
- \alias doch='sudo mksh -c "$(\builtin fc -ln -1)"'
- \command -v rot13 >/dev/null || \alias rot13='tr \
-     abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ \
-@@ -600,4 +601,8 @@
+-\\builtin alias ls=ls l='ls -F' la='l -a' ll='l -l' lo='l -alo'
+-\: "${HOSTNAME:=$(\\builtin ulimit -c 0; \\builtin print -r -- $(hostname \
+-    2>/dev/null))}${EDITOR:=/bin/ed}${TERM:=vt100}${USER:=$(\\builtin ulimit \
++\\builtin alias ls=ls l='ls -F' la='l -a' ll='l -l' lo='l -al'
++\: "${HOSTNAME:=$(</proc/sys/kernel/hostname)} = *([     ]|localhost) && \
```

Comment:
This line is perfect nonsense and caused by a bogus merge. It **needs to be fixed** (well, the entire hunk, likely).